modules/Microsoft.Storage/storageAccounts/localUsers/deploy.bicep

```diff
@@ -0,0 +1,68 @@
+@maxLength(24)
+@description('Conditional. The name of the parent Storage Account. Required if the template is used in a standalone deployment.')
+param storageAccountName string
+
+@description('Required. The local username to be used for SFTP Authentication.')
+param name string
+
+@description('Optional. Indicates whether shared key exists. Set it to false to remove existing shared key.')
+param hasSharedKey bool = false
+
+@description('Required. Indicates whether ssh key exists. Set it to false to remove existing SSH key.')
```

Comment:
In the description field only, please change `ssh` to `SSH` . Applicable to other sections this document

Reply:
Corrected in all sections